blog/2023-01-18-free-software-localization.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: free-software-localization
+
+title: "Free software localization with the Tolgee platform"
+
+authors: [mchalupnikova]
+
+image: "/img/blog/free-localization/painless_localization.png"
+
+description: "With Tolgee localization platform and its own integration libraries, you can quickly and easily translate your software into multiple languages, allowing you to reach a truly global market."
+
+tags: [localization, i18n, translation]
+---
+Overall, localization is an important consideration in software development, as it allows products to be more widely adopted and used by a global market. In addition, it improves the user experience for non-native speakers, builds a positive reputation for the company, and increases its revenue and growth. Unfortunately, software localization has been a source of difficulty for developers, with the challenge of creating and managing localization strings, integrating with external libraries, and more. 
+
+![painless localization](/img/blog/free-localization/painless_localization.png)
+
+<!--truncate-->
+
+_Imagine having a single tool to handle all these tasks, making the translation process much more manageable. A tool that is available for free and which you can self-host._ **That’s when Tolgee comes into the game.** 
+
+## Tolgee platform: what is it, and how can it help with localization?
+
+Tolgee software localization platform is a **free and open-source online tool** that helps software developers localize their products for international markets. Tolgee provides a platform that makes localization easy and affordable with an **easy-to-use interface** for managing translations and supporting all languages. It is constantly being updated with new features and improvements, making it an essential tool for anyone looking to localize their software products. In addition, it provides many great features that make the app localization a piece of cake, such as automatic or machine translation. 
+
+## There are many i18n libraries out there. So why should I use Tolgee?
+
+Tolgee can do a lot more than just traditional i18n library. It doesn't only provide the different strings for different languages, but it also enables you to **change the strings directly in the app you are currently working on**. It also allows your colleagues without coding knowledge to translate/modify/fix strings in your app: even your grandma, aunt, or friend from studies abroad can do that.
```

Comment:
**It not only** provides different strings for different languages, but it also enables you to change the strings directly in the app you are currently working on.

Reply:
Even in production environment, when your app is deployed. I would add a video here.